enhancements/api-review/custom-tags-aws.md

```diff
@@ -178,377 +164,3 @@ In the future we will want to introduce a spec field where a customer can specif
 
 ## Alternatives
 
-Proposal for managing user-defined tags in day2 operations is discussed in the following sections.
```

Comment:
@TrilokGeer Does this mean that you do not intend to add day-2 support or the design was found to be incorrect?

Reply:
@patrickdillon Yeah, I was trying to figure out what the ambiguity was, is it a change in requirements and they don't need day2 support of tags or did they hit a technical constraint? In either case, my suggestion would be to keep the section and specify why we will not be using it. It is in the "Alternatives" section anyways. Removing means we would lose all the investigation/design done here. My 2 cents.